aiida_pseudo/cli/install.py

```diff
@@ -108,9 +110,27 @@ def download_sssp(
     from aiida_pseudo.groups.family import SsspFamily
     from .utils import attempt
 
-    url_sssp_base = 'https://legacy-archive.materialscloud.org/file/2018.0001/v4/'
-    url_archive = f"{url_sssp_base}/{SsspFamily.format_configuration_filename(configuration, 'tar.gz')}"
-    url_metadata = f"{url_sssp_base}/{SsspFamily.format_configuration_filename(configuration, 'json')}"
+    url_prefix = 'https://staging-invenio.materialscloud.org/record/file?filename='
+    url_suffix = '&parent_id=19'
+
+    url_version_yaml = f"{url_prefix}versions.yaml{url_suffix}"
+
+    with attempt('downloading patch versions information... ', include_traceback=traceback):
+        response = requests.get(url_version_yaml)
+        response.raise_for_status()
+        version_mapping = yaml.load(response.content, Loader=yaml.FullLoader)
```

Comment:
Probably want to use `SafeLoader`. I take it the actual version mapping should just contain text and base types, so the `SafeLoader` should be able to load it. This will protect us from arbitrary code execution should the URL ever be hijacked.
```suggestion
        version_mapping = yaml.load(response.content, Loader=yaml.SafeLoader)
```